openvpn/common/hexstr.hpp

```diff
@@ -180,7 +180,7 @@ namespace openvpn {
 	dest.push_back((high<<4) + low);
       }
     if (i != len)
-      throw parse_hex_error(); // straggler char      
+      throw parse_hex_error(); // straggler char
```

Comment:
Sorry for that, I guess Atom fixed this part for me and I haven't noticed that. Should I fix it?